eslint.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = tseslint.config(
 			"lib",
 			"node_modules",
 			"pnpm-lock.yaml",
+			".release-it.js",
```

Comment:
If this is to exist, I'd rather it be linted with `allowDefaultProject`.

Reply:
I'm ok with that, but then I think we should turn some of these rules off for js files (or at least this js file)

![screenshot of eslint ts violations](https://github.com/user-attachments/assets/b5d31df6-46e2-44c5-b54b-b4e253b85192)

Reply:
Or add proper types? I put a `/** @param {{ changelog: string }} context */` atop the `releaseNotes`, change the `groupTitles` loop to a `for (const [type, typeTitle] of Object.entries(groupTitles)`, and most were fixed.

I suspect there's a feature request to be filed upstream in `release-it` to expose a type we can slap on the whole `module.exports`. I think `@type {import("release-it").Config}` would be it, but the `releaseNotes` function doesn't mention the `context` param:

https://github.com/release-it/release-it/blob/897fc3133b9de4107aef7ebb6a120fece174a773/types/config.d.ts#L109

WDYT?